_common/KeymanHosts.php

```diff
@@ -110,7 +110,7 @@ function __construct() {
         $this->s_keyman_com = "{$site_protocol}s.keyman.com{$site_suffix}";
         $this->api_keyman_com = "{$site_protocol}api.keyman.com{$site_suffix}";
         $this->help_keyman_com = "{$site_protocol}help.keyman.com{$site_suffix}";
-        $this->downloads_keyman_com = "{$site_protocol}downloads.keyman.com{$site_suffix}";
+        $this->downloads_keyman_com = "https://downloads.keyman.com";
```

Comment:
Is this _common change to apply to all the other sites?

Reply:
um, no, I was checking something. I'll restore it.

Reply:
Yeah, I did something like that as well when testing.